MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -611,10 +619,10 @@ private void AddMetadataPlugins<T>(List<MetadataPlugin> list, T item, LibraryOpt
             var providers = GetMetadataProvidersInternal<T>(item, libraryOptions, options, true, true).ToList();
 
             // Locals
-            list.AddRange(providers.Where(i => i is ILocalMetadataProvider).Select(i => new MetadataPlugin
+            list.AddRange(providers.Where(i => i is IMetadataProvider && i is not IRemoteMetadataProvider && i is not ILocalMetadataProvider && i is not ICustomMetadataProvider).Select(i => new MetadataPlugin
```

Comment:
Doesn't this kill off the `ILocalMetadataProvider` derived classes that the old code explicitly included?

Also seems very fragile to test for a (limited) set of `is not` instead of explicitly listing the marker interfaces we _want_ to include.